docs/docs/dev_docs/getting_started/noir_contracts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Setting up Aztec.nr Contracts
+---
+
+## Introduction
+
+This guide explains the necessary set up required to write your own contract using the Aztec.nr library and deploy it on the sandbox
+
+:::info Prerequisite reading
+If you haven't read [Aztec Sandbox](./sandbox.md) and [Noir](./noir.md), we recommend going there first.
+:::
+
+### Dependencies
+1. You will need the Noir build tool `nargo`, which you can install via [`noirup`](https://github.com/noir-lang/noirup). Make sure you install the `aztec` version of nargo:
```

Comment:
nit: I would alter this section such that the bullet points are supported by sentences, rather than being sentences.

1. Nargo
_nargo install instructions ..._

2. Sandbox
_Sandbox installation instructions / link_

3. Starter repo
_Link to starter repo_